website/docs/d/oracle_autonomous_database_regular.html.markdown

```diff
@@ -0,0 +1,156 @@
+---
+subcategory: "Oracle"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_oracle_autonomous_database"
+description: |-
+  Gets information about an existing Autonomous Databases.
+---
+
+# Data Source: azurerm_oracle_autonomous_database
+
+Use this data source to access information about an existing Autonomous Databases.
+
+## Example Usage
+
+```hcl
+data "azurerm_oracle_autonomous_database" "example" {
+  name                = "existing"
+  resource_group_name = "existing"
+}
+
+output "id" {
+  value = data.azurerm_oracle_autonomous_database.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of this Autonomous Databases.
```

Comment:
Can you please update `Databases` -> `Database` everywhere in this doc since it refers to a singular database rather than multiple

Reply:
Done, thanks!